### DISCUSSION TITLE
How to currently use the randomizer?

### DISCUSSION THREAD

**Discussion by DemonGamingLP** [phst-randomizer/ph-randomizer#234] [Q&A] [answered]
Ignoring the fact it is early in development and most likely not working fully, is it still possible to use it currently? if yes, could you provide a short guide for it?

**Comment by mike8699** [comment, marked as answer]
Not currently. Once it's in a working state, it will likely be bundled into a single executable that you can just click and run.

**Reply by Niavuag** [reply]
like IN the browser or a downloadable thing ?

**Reply by lunarsoap5** [reply]
It will most likely be a downloadable application
